toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -384,6 +384,13 @@ func PopulateInstallRoot(installChroot *safechroot.Chroot, packagesToInstall []s
 
 	installRoot := filepath.Join(rootMountPoint, installChroot.RootDir())
 
+	if !isRootFS {
+		err = configuration.UpdatePackageRepo(installChroot, config)
```

Comment:
Do we want an option to inject additional repos into a container image?

Reply:
Not enabling this feature for non-ISO builds for now. 

Has re-implemented the logic to only execute this when performing kickstart ISO install